src/installer/managed/Microsoft.NET.HostModel/ComHost/MissingGuidException.cs

```diff
@@ -14,10 +14,15 @@ public class MissingGuidException : Exception
     {
         public MissingGuidException(string typeName)
         {
+#if NET
+            ArgumentNullException.ThrowIfNull(typeName);
+#else
             if (typeName is null)
             {
+
                 throw new ArgumentNullException(nameof(typeName));
-            }
+        }
```

Comment:
nit:
```suggestion
            {
                throw new ArgumentNullException(nameof(typeName));
            }
```

Reply:
I'll have to address this suggestion manually as GitHub does not allow applying suggestions on deleted code.

Reply:
Oh okay there's not really an extra curly bracket, contrary to what the UI is showing. It's just spacing.
I'll wait for the CI to finish first, if no failures look related, I'll manually fix the spacing, then merge. That way we don't have to wait twice for the CI to finish.